Content.Client/Power/APC/UI/ApcMenu.xaml.cs

```diff
@@ -1,25 +1,28 @@
 ﻿using Robust.Client.AutoGenerated;
-using Robust.Client.UserInterface.CustomControls;
 using Robust.Client.UserInterface.XAML;
+using Robust.Client.GameObjects;
 using Robust.Shared.IoC;
 using System;
 using Content.Client.Stylesheets;
 using Content.Shared.APC;
 using Robust.Client.Graphics;
+using Robust.Client.UserInterface.Controls;
 using Robust.Shared.GameObjects;
 using Robust.Shared.Localization;
 using Robust.Shared.Maths;
+using FancyWindow = Content.Client.UserInterface.Controls.FancyWindow;
 
 namespace Content.Client.Power.APC.UI
 {
     [GenerateTypedNameReferences]
-    public sealed partial class ApcMenu : DefaultWindow
+    public sealed partial class ApcMenu : FancyWindow
     {
-        public ApcMenu(ApcBoundUserInterface owner)
+        public ApcMenu(ApcBoundUserInterface owner, ClientUserInterfaceComponent component)
         {
             IoCManager.InjectDependencies(this);
             RobustXamlLoader.Load(this);
 
+            EntityView.Sprite = IoCManager.Resolve<IEntityManager>().GetComponent<SpriteComponent>(component.Owner);
```

Comment:
use a [Dependency] attribute instead

Reply:
Sure thing, let me see what I can do about the toggle.